src/app/components/SplitButton.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  // props
+  export let colorLeft: string = "blue"; // optional
+  export let colorRight: string = "grey"; // optional
+  export let isOutlined: boolean = false; // optional
+  export let hasDropShadow: boolean = false; // optional
+  export let size: string = "medium"; // optional
+  export let leftText: string;
+  export let rightText: string;
+  export let data_cy_left: string;
+  export let data_cy_right: string;
+  export let onClickLeft: (ev: MouseEvent) => void;
+  export let onClickRight: (ev: MouseEvent) => void;
+</script>
+
+<style>
+  .split-button-layout {
+    display: inline-flex;
+    padding: 3%;
+  }
+  button {
+    margin: 0em;
+    border-radius: 0rem;
+    float: left;
+    width: 10rem;
+  }
```

Comment:
Why do you have both `inline-flex` and `float` in these declarations? I think the same layout can be achieved by using `display: flex` and removing the `float` declaration.

Also, please follow this document! https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-omit-the-unit-from-zero